<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab3_widget</class>
 <widget class="QWidget" name="tab3_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_send">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_infotime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通知截止时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_infocount">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通知间隔时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hour">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linehour">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_link">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_linkplace">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_img">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>插入图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ing">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_img2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>插入图片2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_img3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>插入图片3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
